Actions/WorkflowInitialize/WorkflowInitialize.ps1

```diff
@@ -9,18 +9,36 @@ try {
     . (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-Helper.ps1" -Resolve)
     . (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-TestRepoHelper.ps1" -Resolve)
 
-    $ap = "$ENV:GITHUB_ACTION_PATH".Split('\')
+    # $ENV:GITHUB_ACTION_PATH is
+    #                                      -4       -3         -2       -1
+    # linux:            /home/runner/work/owner/AL-Go-Actions/branch/WorkflowInitialize
+    # windows:  C:\Users\runneradmin\work\owner\AL-Go-Actions\branch\WorkflowInitialize
+    #
+    # For Preview (or Direct AL-Go development branch)
+    #                                      -5    -4     -3     -2       -1
+    # linux:            /home/runner/work/owner/AL-Go/branch/Actions/WorkflowInitialize
+    # windows:  C:\Users\runneradmin\work\owner\AL-Go\branch\Actions\WorkflowInitialize
+    $ap = "$ENV:GITHUB_ACTION_PATH".Split([System.IO.Path]::DirectorySeparatorChar)
     $branch = $ap[$ap.Count-2]
     $owner = $ap[$ap.Count-4]
-
-    if ($owner -ne "microsoft") {
+    if ($branch -eq 'Actions' -and $owner -eq 'AL-Go') {
```

Comment:
I'm not sure I understand this check. Could you add a small comment or just explain it in the reply to this comment :)

 Shouldn't `$ap[$ap.Count-2]` always return Actions for example?

Reply:
See comments above - $ap[$ap.count-2] will be the Actions subfolder if using direct AL-Go or preview.
It will be the branch if pointing to AL-Go-Actions